packages/openneuro-components/src/button/Button.tsx

```diff
@@ -14,7 +14,8 @@ export interface ButtonProps {
   color?: string
   imgSrc?: string
   iconSize?: string
-  buttonClass?: string
+  className?: string
+  children
```

Comment:
The type for children is `React.ReactNode` if it can take any arbitrary child component tree.

Reply:
I believe I found all the type declarations and changed them to children: React.ReactNode